app.config.js

```diff
@@ -0,0 +1,21 @@
+import * as dotenv from "dotenv";
+import * as fs from "fs";
+
+const ENV = process.env.APP_ENV || "development";
+const envFile = `.env.${ENV}`;
+
+if (fs.existsSync(envFile)) {
+  dotenv.config({ path: envFile });
+}
+
+export default {
+  name: "RDSApp",
+  slug: "rds-app",
+  scheme: "rdsapp" ,
+  version: "1.0.0",
+  extra: {
+    apiUrl: process.env.EXPO_PUBLIC_API_URL,
+    apiKey: process.env.EXPO_PUBLIC_API_KEY,
```

Comment:
- how are we handling the case for which there is no API URL or API key added in the env?

Reply:
- how're we validating if the required env variables are present?